### ISSUE TITLE
Typo in "Environments"

### ISSUE THREAD

**Issue by jzadra** [hadley/adv-r#1620]
I think there is a typo/disagreement between the text and the image in the environments page.

The text shows `b` getting `d`, however the image shows that `a` now has the value of `d`.
```
The objects don’t live in the environment so multiple names can point to the same object:

e$b <- e$d
```
![image](https://user-images.githubusercontent.com/25590353/84544276-3289af00-acba-11ea-805b-f4ab4395fc26.png)

**Comment by demar01** [comment]
I agree. I think there are few typos in that section. Shouldn't "a" always point to false and "b" get the value of d? 

![image](https://user-images.githubusercontent.com/5008157/102141416-b20a5d80-3e58-11eb-9410-e25b588a14d8.png)
